app/main.py

```diff
@@ -1,3 +1,52 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: ("Distance", float, int)) -> "Distance":
```

Comment:
You can import annotations from __future__ module to omit using quotes on class name typing.
Also when you declare types in parenthesis with commas , those types are considered as individual arguments to type hints. You should use `|` symbol or use `Union` from typing to show that any of those types is possible input